data/upgrades.js

```diff
@@ -919,6 +919,9 @@
     "limited": true,
     "points": 4,
     "text": "<strong>Energy:</strong> You may spend 1 energy to choose 1 friendly ship at Range 1-2. Acquire a target lock, then assign the blue target lock token to the chosen ship.",
+    "size": [
+      "huge"
+    ],
```

Comment:
Most of the cards you changed are titles. These card actually do not say "_Huge ship only_" and are therefor _not_ limited to Huge ships but limited to a specific ship. So I don't think it is correct to set a base size restriction.